c/securityService.c

```diff
@@ -1063,6 +1063,14 @@ static void respondToProfileGET(ClassMgmtCommonParms *commonParms,
 
   zowelog(NULL, LOG_COMP_ID_SECURITY, ZOWE_LOG_DEBUG2, "%s begin\n", __FUNCTION__);
 
+  if (strlen(commonParms->className) > 0) {
+    respondWithError(response, HTTP_STATUS_FORBIDDEN,
+                     "non standard class not allowed");
```

Comment:
In my opinion, this configuration detail (that Zowe security has been configured with a non-standard SAF Class name) should be logged, but Not returned to the browser client.

Reply:
@rocketjared by non-standard class the message means any class except for the `default-class` value passed via the REST request. No configuration details are exposed. Even if you pass the same explicit class name as the one in the config, you'll get this error.

Reply:
As a general rule, web servers should avoid lacing 403 responses with details about why the request is being forbidden.  If details actionable by the system administrator are being placed in the Zowe log, what is the positive case for returning this detail to the client?

Reply:
@rocketjared initially just for development. I agree, it's better to err on the side of caution. I've updated the PR.